frontend/src/visualization/filtering/FilterEntityType.tsx

```diff
@@ -66,7 +63,7 @@ function fetchEdgeTypeFilterModel(
   return filterService.getEdgeTypeFilterModel(edgeName, cancellation);
 }
 
-const EntityFilterElement = (props: {
+const FilterEntityType = (props: {
```

Comment:
It's not a Type, it's a component

Reply:
the EntityType refers to a neo4j EntityType, so for example a Person in the testing dump

Reply:
Maybe reverse the name and call it `EntityTypeFilter`, then it's clear.